pkg/cmd/roachtest/tests/chaos.go

```diff
@@ -159,9 +159,11 @@ func (ch *Chaos) Runner(
 				l.Printf("restarting %v (chaos is done)\n", target)
 				startOpts := option.DefaultStartOpts()
 				startOpts.RoachtestOpts.Worker = true
+				ch.sendEvent(ChaosEventTypePreStartup, target)
```

Comment:
@otan are these going to confuse the DRT tests?

Reply:
Maybe but if you didn't turn the chaos on it doesn't matter.